Source/WTF/Scripts/GeneratePreferences.rb

```diff
@@ -188,11 +190,11 @@ def upcast
   end
 
   def ephemeral?
-    %w{ unstable internal testable }.include? @status
+    %w{ unstable testable }.include? @status
   end
 
   def defaultOverridable?
```

Comment:
While you're here, would you mind renaming this to `defaultsOverridable?`, too? It should be only used in one template.

Reply:
Sure! And it looks like I need to add the casting stuff we used elsewhere. I’ll upload a new patch in the morning.

Reply:
Actually, I actually just need to restore the Uint32 version of the setting method that was removed in [258448@main](https://commits.webkit.org/258448@main)